src/screens/timeStamping/EventEdition/index.tsx

```diff
@@ -5,32 +5,13 @@ import Events from '../../../api/Events';
 import { addTime, changeDate, dateDiff, formatDate, getEndOfDay, isBefore } from '../../../core/helpers/dates';
 import FeatherButton from '../../../components/FeatherButton';
 import NiErrorMessage from '../../../components/ErrorMessage';
-import ExitModal from '../../../components/modals/ExitModal';
+import ConfirmationModal from '../../../components/modals/ConfirmationModal';
+import EventDateTimeEdition from '../../../components/EventDateTimeEdition';
 import NiPrimaryButton from '../../../components/form/PrimaryButton';
-import styles from './styles';
 import { COPPER, COPPER_GREY } from '../../../styles/colors';
 import { ICON } from '../../../styles/metrics';
-import { EventType } from '../../../types/EventType';
-import { NavigationType } from '../../../types/NavigationType';
-import EventDateTimeEdition from '../../../components/EventDateTimeEdition';
-
-export type ModeType = 'date' | 'time';
-
-interface EventEditionProps {
-  route: { params: { event: EventType } },
-  navigation: NavigationType,
-}
-
-export interface EventEditionStateType {
-  startDate: Date,
-  endDate: Date,
-  start: boolean,
-}
-
-export interface EventEditionActionType {
-  type: string,
-  payload?: { date?: Date, mode?: ModeType, start?: boolean },
-}
+import styles from './styles';
+import { EventEditionActionType, EventEditionProps, EventEditionStateType } from './types';
```

Comment:
Proposition de gestion des types : on pouvait aussi les gérer via une fichier de déclaration de type mais j'ai l'impression que c'est trop poussé pour ce qu'on veut faire (sur les articles ils parlent plutôt de le faire quand on créer des packages npm) : 
- https://dev.to/kowalevski/what-are-type-declaration-files-in-typescript-53dd
- http://definitelytyped.org/guides/best-practices.html